lib/src/main/java/com/bq/appupdateshelper/FakeAppUpdatesHelper.java

```diff
@@ -63,26 +63,6 @@ public void setUpdateAvailable(int availableVersionCode) {
         fakeAppUpdateManager.setUpdateAvailable(availableVersionCode);
     }
 
-    /**
-     * Returns which type of update is partially allowed or null if no update is partially allowed.
-     */
-    @Nullable
-    public Integer getPartiallyAllowedUpdateType() {
-        return fakeAppUpdateManager.getPartiallyAllowedUpdateType();
-    }
-
-    /**
-     * In case the update is only partially allowed, which means that not all in-app update types are allowed by policy,
-     * the type of update which is exclusively allowed can be set here.
-     * <p>
-     * Also sets the error code to {@link InstallErrorCode.NO_ERROR_PARTIALLY_ALLOWED} to enable the partial error behaviour.
-     *
-     * @param allowedUpdate one of {@link AppUpdateType.FLEXIBLE} or {@link AppUpdateType.IMMEDIATE}, or null
-     */
-    public void setPartiallyAllowedUpdateType(@Nullable @AppUpdateType Integer allowedUpdate) {
-        fakeAppUpdateManager.setPartiallyAllowedUpdateType(allowedUpdate);
-    }
```

Comment:
This methods disapear on last version :/

Reply:
https://developer.android.com/reference/com/google/android/play/core/release-notes

There's no info about it...

Reply:
They may have removed it without warning 🤔?

Reply:
Weird..

What was the function of that methods?

Reply:
Seems like they did something with partial updates in `v1.7.3`, so probably that's where the methods were removed: https://developer.android.com/reference/com/google/android/play/core/release-notes#1-7-3

If I remember correctly this allowed you to set the kind of update that the app supports, either `FULL` or `FLEXIBLE`

Reply:
But the kind of update you set it up on `onUpdateStatusResult` with the `requestCode` param. SO maybe this is not necessary anymore

Reply:
Yeah, it may not be needed, so leave it deleted, it's OK